Backpack/Theme/Classes/BPKDefaultThemeContainer.h

```diff
@@ -15,11 +15,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
I always hear @matthewdavidson's voice in my head saying _"Let the code breathe!"_ when I see these 👍 😂